client/src/actions/index.js

```diff
@@ -42,7 +42,7 @@ export const login = (email, password) => async (dispatch) => {
         }
 
         dispatch({ type: IS_LOADING, payload: false });
-
+        dispatch({type: REQUEST_SUCCESS, payload: true})
```

Comment:
If we were to dispatch this action, the action "REQUEST_SUCCESS" expects and object with a "msg" field.

dispatch({ type: REQUEST_SUCCESS, payload: { msg: "Login Successful!" });

Like I said in the discussion of this PR, this isn't needed since the user is redirected. Same with registration, which can be changed.